pages/common/docsify.md

```diff
@@ -0,0 +1,24 @@
+# docsify
+
+> Initialize and serve markdown documentation.
+> More information: <https://cli.docsifyjs.org>.
+
+- Initialize a new documentation in the current directory:
+
+`docsify init`
+
+- Initialize a new documentation in the specified directory:
+
+`docsify init {{path/to/directory}}`
+
+- Serve local documentation on `localhost:3000` with live reload:
+
+`docsify serve {{path/to/directory}}`
+
+- Serve local documentation on `localhost` at the specified port:
+
+`docsify serve --port {{port}} {{path/to/directory}}`
```

Comment:
```suggestion
`docsify serve --port {{80}} {{path/to/directory}}`
```

Feel free to ignore this one if it is incorrect, however I feel this makes it a little easier for readers